west.yml

```diff
@@ -40,7 +40,7 @@ manifest:
     - name: fw-nrfconnect-zephyr
       path: zephyr
       west-commands: scripts/west-commands.yml
-      revision: 7e64ac737daead12d2dbdda5daec04188243a81c
+      revision: f69d8d61a934533de2dfc5c3ca915b82910acd8d
```

Comment:
why a sha and not `pull/166/head` while the other PR is still in review? the final sha will be different depending on who commits it.

Reply:
Indeed you should use `pull/166/head`. The PR branch lies on your fork which is a different remote - `west` cannot find it.

Reply:
Right, fixed that now. @rlubos @mbolivar

Reply:
pull/166/head is a 'branch' name that can move (not good for traceability) .  I think the policy is west  manifests should have sha's or tag's( which don't move)
@carlescufi   input?